elliottlib/bzutil.py

```diff
@@ -177,8 +177,22 @@ def get_remotelinks(self, bugid):
                 linkdict[link.relationship] = link.object.url
         return linkdict
 
-    def update_bug(self, bugid, status):
-        self._client.transition_issue(self._client.issue(bugid), status)
+    def update_bug_status(self, bugid, target_status, comment_for_release=None, noop=False):
```

Comment:
I would actually like these methods to be in JIRABug and BugzillaBug classes
so you can just call `bug.update_status()` and `bug.add_comment()`

Reply:
for bugzilla bug it can work as it used to be, but for jirabug, update status need init jira client, it's not a self function for jira issue